testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -6680,3 +6688,1393 @@ fn test_flash_block_skip_tenure() {
 
     channel.stop_chains_coordinator();
 }
+
+#[test]
+#[ignore]
+fn exit_at_rc_integration_test() {
+    if env::var("BITCOIND_TEST") != Ok("1".into()) {
+        return;
+    }
+
+    let spender_sk = StacksPrivateKey::new();
```

Comment:
this gets copied around but could we make a reusable util function that would return triples of (sk, addr, stacks_addr)?